app/assets/stylesheets/administrate/application.scss

```diff
@@ -10,3 +10,5 @@
 @import "library/**/*";
 @import "base/**/*";
 @import "components/**/*";
+
+@import "administrate/customize";
```

Comment:
I have no real preferences on naming, are you asking me to change it?

Make that change, and the test fails.  I think the current directory is first in the search path, so that import will just bring in the placeholder.  I want to force a lookup where the app stylesheets path comes before the gem stylesheets path, and therefore finds the override file, but only if it exists.